Open all selected files instead of just the first

rnpnr opened the pull request:
Hi! Cool plugin! I think that one of the advantages of `nnn` over
`vis-menu` is that it allows you to select multiple files. To take
advantage of that I made the plugin open every file that was selected
instead of just the first. This does change the plugin to open a new
window even when only one file is selected though. If you are interested
in keeping that behaviour I can rework the patch to have two commands
`:ennn` and `:onnn` where `:ennn` will behave like the plugin currently
does.

rnpnr committed:
avoid string.format on strings

captaingroove commented:
I'm glad you like the plugin and thanks for the patch!

What about opening the first file with `e` and all other files with `open`? Then it would need only one `nnn` command which would be consistent with the former behavior when opening only one file. I also noticed, that you probably have a custom `o` command as an abbreviation for `open`, it seems to be not available by default.

rnpnr committed:
export nnn() which returns a table of files

rnpnr committed:
open any additional files in new windows

rnpnr commented:
I guess I don't really use the `e` command in my uses but I can see
why you prefer that behaviour. I added a 3rd commit which exports an
`nnn()` function that returns a table of selected files. I can use it
to add my own prefered command. It might also be useful for doing other
things besides just opening files.

> I also noticed, that you probably have a custom `o` command

That should be default. If it doesn't work for you I need to look into
why its inconsistent. In any case I changed it to `open`.

captaingroove commented:
Very nice solution. Thanks also for updating the docs!
